modules/porous_flow/doc/content/source/dirackernels/PorousFlowPointEnthalpySourceFromPostprocessor.md

```diff
@@ -0,0 +1,24 @@
+# PorousFlowPointEnthalpySourceFromPostprocessor
+
+`PorousFlowPointEnthalpySourceFromPostprocessor`
+implements an enthalpy point source that adds fluid at a
+mass flux rate (computed by a postprocessor) and specified temperature (computed by a postprocessor).
```

Comment:
Mentioning the requirement to use the identical fluid property object makes sense to me.  It seems misleading to mention the materials, since they are not really needed in this Dirac kernel setup... Unless you have some other reason to mention them.